source/_integrations/python_script.markdown

```diff
@@ -90,3 +90,13 @@ turn_on_light:
 ```
 
 For more examples, visit the [Scripts section](https://community.home-assistant.io/c/projects/scripts) in our forum.
+
+## Services
+
+Available services: `reload`.
+
+#### Service `python_script.reload`
```

Comment:
This should have been a level 3 heading.

@boc-the-git Please open up a PR to correct this.

Reply:
Will do @frenck . I did base that on something, so I'll try to also find where that was and fix it too.. just not sure I remember.